package.json

```diff
@@ -19,10 +19,11 @@
     "url": "https://github.com/quintype/quintype-js/issues"
   },
   "homepage": "https://github.com/quintype/quintype-js#readme",
-  "dependencies": {
+  "peerDependencies": {
```

Comment:
@ThisIzKp  https://nodejs.org/en/blog/npm/peer-dependencies/ - they say we should not lock the peer dependency down to a specific version and instead make it as relaxed as possible. Check the example at the end.